generator/src/test/java/com/scottlogic/deg/generator/walker/reductive/ReductiveDecisionTreeReducerTests.java

```diff
@@ -296,15 +296,15 @@ public void shouldReturnNullIfTreeReductionInvalidatesTheTree(){
     @Test
     public void shouldReturnNullIfALowLevelDecisionIsInvalidated(){
         AtomicConstraint field3IsGreaterThan4 = new IsGreaterThanConstantConstraint(field3, 4, Collections.emptySet());
-        TreeConstraintNode field2GreaterThan2Option = new TreeConstraintNode( //expect this to be removed
+        TreeConstraintNode field2GreaterThanOrEqualTo2Option = new TreeConstraintNode( //expect this to be removed
             Collections.singletonList(isGreaterThanOrEqualTo2),
             Collections.singletonList(
                 new TreeDecisionNode(
                     new TreeConstraintNode(field3IsGreaterThan4) //something that isn't yet fixed and itself wouldn't be removed
                 )
             ));
         DecisionNode decision1 = new TreeDecisionNode(
-            field2GreaterThan2Option, //field2 > 2
+            field2GreaterThanOrEqualTo2Option, //field2 > 2
```

Comment:
Minor comment: clarify the comment here

```suggestion
            field2GreaterThanOrEqualTo2Option, //field2 >= 2
```

Reply:
thanks, changed.